re/service.go

```diff
@@ -41,9 +53,52 @@ func New(secret string) Service {
 	}
 }
 
-func (ks *reService) Ping(secret string) (string, error) {
-	if ks.secret != secret {
-		return "", ErrUnauthorizedAccess
+type info struct {
+	Version       string `json:"version"`
+	Os            string `json:"os"`
+	UpTimeSeconds int    `json:"upTimeSeconds"`
```

Comment:
Probably better``json:"up_time_seconds"``

Reply:
This is due to kuiper internals and is used to deserialize information fetched from kuiper.